Library/Formula/docker-machine.rb

```diff
@@ -4,8 +4,8 @@ class DockerMachine < Formula
   desc "Create Docker hosts locally and on cloud providers"
   homepage "https://docs.docker.com/machine"
   url "https://github.com/docker/machine.git",
-    :tag => "v0.5.6",
-    :revision => "61388e98540321b34f4a27f88df9e7a4443b9ac8"
+    :tag => "v0.6.0",
+    :revision => "e27fb87286cb8e7454183ce46a6e1e84b31965e9"
   revision 1
```

Comment:
Line 9 (revision 1) should be removed as there is is a minor version change. Revision only gets added when the version doesn't change. See Formulae Revisions in https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md